Feat aligned section

JamesVictor-O opened the pull request:
# 📝 Pull Request Title
"We are aligned with" section for technology alignment

## 🛠️ Issue
- Closes #17 

## 📖 Description
-This PR introduces the "We are aligned with" section, which provides a quick visualization of the technologies we align with. The section is fully responsive and optimized for performance, ensuring it loads in under one second.

## ✅ Changes made
✅ Designed and implemented the section with responsive styling
✅ Ensured compatibility across various screen sizes
✅ Optimized assets and layout for fast loading (<1s)

## 🖼️ Media (screenshots/videos)
![Screenshot from 2025-02-21 08-45-29](https://github.com/user-attachments/assets/6ec066b2-09f9-4994-afba-81105f3cc59d)
![Screenshot from 2025-02-21 08-45-50](https://github.com/user-attachments/assets/322dc0f7-8807-49f9-a6aa-37513d526f8d)



## 📜 Additional Notes
 Verified responsiveness on mobile, tablet, and desktop
 Measured load time to ensure it meets performance standards
Would love feedback on any improvements or additional refinements

diegoTech14 commented on apps/frontend/components/AlignedWith/AligendWith.tsx:
```diff
@@ -0,0 +1,79 @@
+  ];
+
+  return (
+    <div className="w-full h-screen md:h-[280px] bg-[#101010] text-[#FFFFFF] py-20 px-4 md:px-8">
```
Could you please use the css variables, for example instead use bg-[#101010] you can use bg-secondary

diegoTech14 commented:
The responsive is looks good but there are some resolutions where the content is not aligned:
![image](https://github.com/user-attachments/assets/e90e76cd-b153-4040-bdba-2a059744e8be)

diegoTech14 commented on apps/frontend/components/AlignedWith/AligendWith.tsx:
```diff
@@ -0,0 +1,79 @@
+          <h2 className="text-4xl font-bold text-white mb-4">
+            We are aligned with
+          </h2>
+          <p className="text-zinc-400 text-base  md:text-lg max-w-2xl mx-auto">
```
Please, change the color text of the `p` tag from text-zing-400 to text-secondary

JamesVictor-O commented:
@diegoTech14  i implemented the required changes and also aligend the ecosystems in on mobile and tablet view, if there are othere things i should fix let me know

diegoTech14 commented:
Hello @JamesVictor-O could you please make these changes?

1. Change the background color per partner, just use bg-black instead bg-white or hover effect
2. Could you delete the button:      
```
<Link href="/app">
    <Button>Go to App</Button>
</Link> 
```
from the page.tsx please?

The responsive looks amazing, thanks, good job! 🚀

JamesVictor-O commented:
@diegoTech14  thank you for the pointer on what to fix and how to do the, i Changed the background color per partner to bg-black as instructed, removed the button in the page.tsx file and give the patner card a bg-white on hover. thank you.
![Screenshot from 2025-02-22 20-19-58](https://github.com/user-attachments/assets/fa084c7e-6e41-4f6d-846b-ebaf3c62bb2d)

JamesVictor-O commented:
@diegoTech14  i implemented the changes and resloved the conflict, please check it out

diegoTech14 commented:
@JamesVictor-O Thanks for the contribution sir! 🚀
